packages/autorest.typescript/src/transforms/operationTransforms.ts

```diff
@@ -607,3 +610,21 @@ function getMapperForSchema(
     })
   );
 }
+
+function isOperationTenantLevel(operation: Operation) {
+  const subscriptionIdParameter = operation.parameters?.find(param => {
+    return (
+      param.protocol.http?.in === ParameterLocation.Path &&
+      param.language.default.name.toLowerCase() === "subscriptionid" &&
+      param.implementation === "Client"
+    );
+  });
+  if (
+    subscriptionIdParameter ||
+    operation.operationId === "Operations_List" ||
```

Comment:
no, because operations_list and check name availability or check name exist operation don't have a subscriptionId, and if one RP only has this two operations without client level subscriptionId, then it should not be considered as a RP that has tenant level operations.